### ISSUE TITLE
Right stick drifring on wii u gamepad. (JK2)

### ISSUE THREAD

**Issue by NeroTheYoung** [FIX94/Nintendont#1021]
It happens like 40% of the times I move the right stick to the left playing Jedi Knight 2.
It only happens in this game and I can assure it's not my gamepad since other games like Bayonetta and BOTW don't have this issue. 

I've read somewhere that other people were having similar issues with other gamecube games, however none of them was using the gamepad. 

Any help would be much appreciated

**Comment by spudpiggy** [comment]
It sounds like your GamePad is slightly drifting, it's inevitable.
The reason you don't have this issue in other games may be because they use a larger deadzone, and the drift is within that - however it sounds like Jedi Knight 2 (or Nintendont) uses a much smaller deadzone which your drift isn't within - hence you experience drift.
tl;dr your gamepad is probably drifting, but this only becomes problematic with smaller deadzones.

**Comment by NeroTheYoung** [comment]
Is there any way to increase the deadzone in Nintendon't for wii u? 

https://gbatemp.net/threads/gamecube-controller-drift-using-nintendont.566067/ 

The last message there said it was possible but not if it is forwarded in Wii U.  

I'd like to know why is it impossible and if there's any kind of alternative since the .ini file he's referring is nowhere to be found.

**Comment by carnage702** [comment]
> Is there any way to increase the deadzone in Nintendon't for wii u?
> 
> https://gbatemp.net/threads/gamecube-controller-drift-using-nintendont.566067/
> 
> The last message there said it was possible but not if it is forwarded in Wii U.
> 
> I'd like to know why is it impossible and if there's any kind of alternative since the .ini file he's referring is nowhere to be found.

of course its possible but it would affect every single game, making some games basically unplayable due to them using like walk and run and you would only be able to run for instance, because the deadzone ate all the walk values and such, the deadzone is already preety damn huge on wiiu gamepad anymore and starts breaking stuff.

1 that was an issue with gc controller on wiiugc adapter and has been solved, it has nothing to do with wiiu gamepad.
2 wiiu gamepad and bt controllers(the ones that dont connect trough cable) dont have ini files they are all hardcoded into nintendont so you cant simply change them,Also deadzone is not on the inis at all.
3 there is no difference from nintendont on wii or wiiu so what changes happen on wiiu would happen on vwii and vice versa.

Te the deadzone for the wiiu gamepad you would need to modify nintendont code and then recompile it there is no other way.
